types/account.ts

```diff
@@ -255,31 +344,42 @@ export interface AccountLimits {
     } & UnimplementedFields
 }
 
-export type PatchAccountRequest = PatchDepositAccountRequest
+export type PatchAccountRequest = PatchDepositAccountRequest | PatchCreditAccountRequest
 
 export interface PatchDepositAccountRequest {
     accountId: string
 
     data: {
         type: "depositAccount"
         attributes: { 
-            tags?: object
+            tags?: Tags
             depositProduct?: string
         }
     }
 }
 
+export interface PatchCreditAccountRequest {
+    accountId: string
+
+    data: {
+        type: "creditAccount"
+        attributes: { 
+            tags?: Tags
+            creditLimit?: number
+        }
+    }
+}
+
 export interface BatchAccount {
     type: "batchAccount"
     id: string
     attributes: {
-        createdAt: string
-        name: string
-        routingNumber: string
-        accountNumber: string
-        balance: number
-        hold: number
-    }
+        depositProduct?: string
+        routingNumber?: string
+        accountNumber?: string
+        closeReason?: CloseReason
+        maskedAccountNumber?: string
+    } & Pick<BaseAccountAttributes, "createdAt" | "name" | "status" | "balance" | "hold" | "currency" | "available" | "fraudReason"> & UnimplementedFields
```

Comment:
Would `Omit` be better here?
Also missing updatedAt from the pick, not sure it's documented enough for batch accounts though